docs/admin/deployment/tor_pow.rst

```diff
@@ -0,0 +1,51 @@
+Tor Proof-of-Work Defense on the *Source Interface*
+===================================================
+
+The SecureDrop *Source Interface* is served as an onion service with an
+``.onion`` URL, requiring Tor Browser to access it over the Tor network.  Tor is
+sometimes targeted for denial-of-service (DoS) attacks that can `slow down the
+Tor network as a whole <https://blog.torproject.org/tor-network-ddos-attack/>`_
+as well as burden individual onion services, including SecureDrops.
+
+Tor now includes a `proof-of-work (PoW) defense
+<https://community.torproject.org/onion-services/ecosystem/technology/pow/>`_
+against denial-of-service attacks that can be turned on for individual onion
+services.  As of SecureDrop 2.9.0, new SecureDrops have this feature enabled by
+default, and we encourage all SecureDrop administrators to turn it on for their
+instances.  While this measure can't speed up the Tor network as a whole if it's
+slow, it can protect your SecureDrop from being attacked specifically; and more
+onion services running with this feature helps improve the resilience of the Tor
+network.
+
+
+Enabling the proof-of-work defense
+----------------------------------
+
+Make sure you have :doc:`installed SecureDrop already
```

Comment:
Is is strictly necessary for users to have already installed SecureDrop, since they will be prompted for this option anyway during the initial `sdconfig` run?

Reply:
Good point.  I borrowed this language from here—

https://github.com/freedomofpress/securedrop-docs/blob/a26b3fb0e492048de93ab7bbe50f6efc5c6967fa/docs/admin/deployment/https_source_interface.rst?plain=1#L143

—where I suppose the same objection applies: you _can_ enable this right out of the gate, from your initial `securedrop-admin {sdconfig,install}`.  The difference is that we're now enabling `securedrop_app_pow_on_source_interface` by default, which we are not doing for `securedrop_app_https_on_source_interface`.

Let me know what you think of d33e5cb5a3ed27f5c4ab113926c980e2b50253c9, which tries to clarify this.

Reply:
Thanks @cfm - I think the changes you made here are excellent!